eclipse-theia/theia
# tree: fix right-clicking tree nodes bug

#### What it does

Fixes: https://github.com/eclipse-theia/theia/issues/12800

This PR fixes the small regression induced by https://github.com/eclipse-theia/theia/pull/12783 where right-clicking a tree node would open it.
#### How to test


1. Open Theia.
2. Go to the `Explorer` view and open a folder.
3. Right-click on a tree node to open the context menu.
4. The selected file should not be opened.
5. Middle-clicking to open tree nodes should still be functional.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
